CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v0.0.26
```

Comment:
I think these are technically breaking changes, but probably not worth bumping a major version for